action_msgs/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>action_msgs</name>
+  <version>0.5.0</version>
+  <description>Messages and service definitions common among all ROS actions.</description>
+  <maintainer email="karsten@osrfoundation.org">Karsten Knese</maintainer>
+  <license>Apache License 2.0</license>
+
+  <author email="jacob@openrobotics.org">Jacob Perron</author>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
+
+  <build_depend>builtin_interfaces</build_depend>
+
+  <exec_depend>builtin_interfaces</exec_depend>
```

Comment:
I think `<depend>builtin_interfaces</depend>` to get all three `build_depend`, `exec_depend`, and `build_export_depend` would make sense because a C  generator would export a header with an include to `builtin_interfaces/msg/Time.h`

Reply:
In general there are only two types of dependencies: the ones for build time and the ones for run time. Internally `exec_depend` and `build_export_depend` do the same thing - they are only named separately since users found it "easier" / more intuitive than `run_depend`.

Anyway if you use both then using `depend` makes sense :+1: